Versão da Sprint 3

Lucas-nepomuceno opened the pull request.

daviiabreu reviewed (approved):
Ótimo, Pablo, ficou sensacional!

Melomm committed:
DroneApp Template

Melomm committed:
feat: Drone App Template

daviiabreu committed:
feat: criei uma pasta

zzaved committed:
feature: adiciona padding e redução

Este commit adiciona o padding e a redução do tamanho da imagem

daviiabreu committed:
feat: adiciona as imagens de fissuras

daviiabreu committed:
feat: adiciona as imagens da fissura

iisabelledantas committed:
feat: adiciona o componente da  mini galeria

Lucas-nepomuceno committed:
Merge pull request #6 from Inteli-College/feat/cria-tela-upload

feat: adiciona o componente da  mini galeria

Lucas-nepomuceno committed:
Merge pull request #5 from Inteli-College/feat/tela-principal

Feat/tela principal

zzaved committed:
feature: adiciona testes

Este commit adiciona os testes entre Sobel, Canny e Ann. A comparação dos trÊs é importante para compreender qual aplicar de fato no refinamento das imagens.

Melomm committed:
feat: Drone app v0.1 (Testing)

zzaved committed:
feature: adiciona código de comparação de filtros

Este commit adiciona o código que busca comparar Canny, Sobel e ANN na aplicação de imagens

zzaved committed:
Merge branch 'feat/ia-agro-lei' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/ia-agro-lei

daviiabreu committed:
feat: preprocess visualization

iisabelledantas committed:
fix: adiciona a funcionalidade de ampliar e excluir fotos na  mini galeria

daviiabreu committed:
feat: folder organization and pipeline base extructure

zzaved committed:
feature: adiciona pipeline para estudos

Conforme combinado com o meu colega de grupo, Davi, iremos modularizar filtros e criar uma pipeline para essa aplicação. Nese commit, estou estudando como funciona a modularização de filtros e criação de uma pipeline

Melomm committed:
feat: Drone App v0.2 (Testing Phase)

Lucas-nepomuceno committed:
refactor: reorganiza as pastas

Lucas-nepomuceno committed:
Merge branch 'feat/cria-tela-upload' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/upload-mini-galeria

Lucas-nepomuceno committed:
feat: adiciona a tela de upload com a mini galeria

zzaved committed:
feature: adiciona modularização de filtros

Este arquivo adiciona a modularização dos filtros a serem aplicados na imagem no pré-processamento na pipeline

zzaved committed:
feature: adiciona vetorização do padding

Este commit adiciona a vetorização do padding nas imagens para serem adicionadas na pipeline

zzaved committed:
Feature: adiciona vetorização

Este commit adiciona a vetorização de componentes para a pipeline e uma pipeline de testes

Lucas-nepomuceno committed:
fix: coloca as paginas como paginas

iisabelledantas committed:
Merge pull request #8 from Inteli-College/feat/upload-mini-galeria

Feat/upload mini galeria

iisabelledantas committed:
feat: adiciona a base do front-end da tela de historico

lupetenazzi committed:
feat: adiciona a função que realiza a busca das imagens no supabase com a coluna app igual a true

esse commit adiciona a função e adiciona uma tentativa falha de teste devido a um erro encontrado em uma outra pagina, para proximos passos a função deve ser alterada de maneira que ela nao execute automaticamente e sim ao ser selecionada por um botão

daviiabreu committed:
feat: pipeline completa com augmentation e filtros integrados com pablo

daviiabreu committed:
Merge pull request #9 from Inteli-College/feat/ia-agro-lei

Feat/ia agro lei

lupetenazzi committed:
fix: arruma funcao de buscar imagens

Lucas-nepomuceno committed:
feat: adiciona tanto a possibilidade de baixar relatorio quanto preview

julia-lika committed:
feat: adiciona a integracao das imagens nos buckets do backend

daviiabreu committed:
feat: ai model swin transformer

lupetenazzi committed:
feat: adiciona o funcionamento do botao e da mini galeria na funcao de carregar imagens pelo servidor

Lucas-nepomuceno committed:
feat: cria a pagina que mostra um preview do relatorio

Lucas-nepomuceno committed:
Merge pull request #10 from Inteli-College/feat/funcao-image

Feat/funcao image

Lucas-nepomuceno committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/integracao-ia

daviiabreu committed:
fix: erros no modelo corrigidos

daviiabreu committed:
fix: linux paths for supercomputer

daviiabreu committed:
fix: import base dir fix

daviiabreu committed:
fix: torch weights fix

Lucas-nepomuceno committed:
feat: escreve o pdf corretamente

julia-lika committed:
Revert "feat: Drone App v0.2 (Testing Phase)"

This reverts commit ac420ac366f5aae94b413db9fd7ed554b94259b7.

daviiabreu committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/ai-model-davi

julia-lika committed:
Voltando ao commit Drone App v0.2 (Testing Phase)

julia-lika committed:
Merge branch 'feat/app-drone' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/app-drone

daviiabreu committed:
feat: modelos treinados

julia-lika committed:
Revert "Merge branch 'feat/app-drone' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/app-drone"

This reverts commit 1dec3747337e09ce76961601c36ef91ed47897e7, reversing
changes made to 5456392da5a31b17b0aab5a2435b78f89bc221ad.

julia-lika committed:
feat: Esvazia SupabaseClient.kt

Melomm committed:
feat: Botão de disconnect e melhoras no decoder

iisabelledantas committed:
feat: adiciona a tela de historico

iisabelledantas committed:
Merge pull request #11 from Inteli-College/feat/preview-historico

Feat/preview historico

daviiabreu committed:
feat: modulariza dentro de src para diferentes modelos em pastas diferentes

Lucas-nepomuceno committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/cria-tela-historico

lupetenazzi committed:
Merge pull request #13 from Inteli-College/feat/ai-model-davi

Feat/ai model davi

Lucas-nepomuceno committed:
feat: adiciona a pagina no botao

Lucas-nepomuceno committed:
Merge pull request #12 from Inteli-College/feat/cria-tela-historico

Feat/cria tela historico

iisabelledantas committed:
fix: corrige falhas no upload da imagem

daviiabreu committed:
feat: integracao da ia do pablo

zzaved committed:
Merge pull request #14 from Inteli-College/feat/integra-ias

feat: integracao da ia do pablo

daviiabreu committed:
feat: interference unificado

daviiabreu committed:
feat: trocando o swin por padrao

zzaved committed:
docs: adiciona estruturação de pastas

Este commit adiciona a estruturação de pastas para serem preenchidas na documentação

iisabelledantas committed:
feat:faz integração inicial

Melomm committed:
Merge pull request #16 from Inteli-College/docs/organiza-documentação

docs: adiciona estruturação de pastas

Lucas-nepomuceno committed:
Merge pull request #15 from Inteli-College/feat/integracao-ia-e-front

feat: interference unificado

Lucas-nepomuceno committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/integracao-ia

Lucas-nepomuceno committed:
Merge branch 'feat/integracao-ia' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into feat/integracao-ia

Melomm committed:
Hotfix: Docs template

Lucas-nepomuceno committed:
feat: integra back e front --aviso os requirements devem ser refeitos

daviiabreu committed:
docs: documentacao do swin transformer

daviiabreu committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into featura/IAgro

Melomm committed:
docs: Introdução e Controle com Streaming

lupetenazzi committed:
docs: adiciona o link de acesso para a apresentação da sprint 2 e 3 na documentação

julia-lika committed:
feat: adiciona a conexão com o banco de dados do supabase

lupetenazzi committed:
docs: adiciona a documentação do escopo da sprint 3 com a definição das equipes responsaveis

Lucas-nepomuceno committed:
fix: ajusta os requirements

lupetenazzi committed:
docs: adiciona previa dos links de relação entre os assuntos da sprint

iisabelledantas committed:
docs: fluxo do usuario no frontend

lupetenazzi committed:
Merge pull request #17 from Inteli-College/feat/integracao-ia

Feat/integracao ia

lupetenazzi committed:
docs: adiciona documentação das funcionalidades complementares do sistema

lupetenazzi committed:
docs: adiciona as correções apontadas pelo gov

Lucas-nepomuceno committed:
fix: permite apenas que swin seja chamado

lupetenazzi committed:
Merge pull request #22 from Inteli-College/feat/integracao-ia

fix: permite apenas que swin seja chamado

lupetenazzi committed:
Merge pull request #18 from Inteli-College/feat/app-drone

feat: adiciona a conexão com o banco de dados do supabase

julia-lika committed:
docs: adiciona documentação de envio de imagens para o supabase

lupetenazzi committed:
docs: arruma o caminho das imagens

lupetenazzi committed:
.

julia-lika committed:
Merge pull request #21 from Inteli-College/docs/fluxo-usuario

Docs/fluxo usuario

Lucas-nepomuceno committed:
fix: ajusta servidor

Lucas-nepomuceno committed:
Merge pull request #24 from Inteli-College/feat/integracao-ia

fix: ajusta servidor

Lucas-nepomuceno committed:
fix: corrige pastas

zzaved committed:
docs: ajusta o ambiente para documentação

Este commit ajusta arquivos para a documentação da sprint

zzaved committed:
docs: ajusta ambiente

Este commit ajusta arquivos do ambiente de documentação

zzaved committed:
docs: adiciona guia de execução

Este commit adiciona o guia de execução da pipeline para rodar os modelos de IA do projeto

zzaved committed:
docs: documenta primeiro modelo

este commit documenta o primeiro modelo de IA

zzaved committed:
docs: adiciona segundo modelo

Este commit adiciona o segundo modelo de IA

zzaved committed:
docs: pipeline unificada

Este commit adiciona a pipeline unificada

daviiabreu committed:
docs: adiciona parte de filtros da ia e correcoes

daviiabreu committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into docs/organiza-documentação

daviiabreu committed:
docs: corrije a parte de pre processamento

daviiabreu committed:
Merge pull request #20 from Inteli-College/docs/escopo

Docs/escopo-sprint3

daviiabreu committed:
Merge pull request #25 from Inteli-College/docs/fix-pastas

fix: corrige pastas

daviiabreu committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into docs/organiza-documentação

daviiabreu committed:
Merge pull request #23 from Inteli-College/docs/app-drone

Docs/app drone

daviiabreu committed:
Merge branch 'develop' of https://github.com/Inteli-College/2025-1B-T12-EC06-G02 into docs/organiza-documentação

zzaved committed:
Merge pull request #26 from Inteli-College/docs/organiza-documentação

Docs/organiza documentação

Lucas-nepomuceno committed:
docs: inclui as instruções de como rodar o sistema

Lucas-nepomuceno committed:
Delete docs/docs/sprint-3/front-end directory

Melomm committed:
Merge branch 'develop' into docs/como-rodar

Melomm committed:
Fix: Package docusaurus dependency

Melomm committed:
Merge pull request #27 from Inteli-College/docs/como-rodar

docs: inclui as instruções de como rodar o sistema

Lucas-nepomuceno committed:
hotfix: valida tudo antes de ir pra main

Melomm reviewed (approved):
Dev para Main, vai com deus gov

Lucas-nepomuceno committed:
hotfix: valida mesmo tudo antes de ir pra main
